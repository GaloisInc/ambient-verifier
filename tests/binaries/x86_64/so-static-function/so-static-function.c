#include "ambient_assert.h"
#include "getone-static-function.h"

int main() {
  ambient_assert(getone() == 1);
  return 0;
}
