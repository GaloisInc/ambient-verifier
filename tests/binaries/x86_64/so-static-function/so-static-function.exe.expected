successful: 0
failed: 0
sharedObjectsDir: tests/binaries/x86_64/so-static-function
